recipes/aioserial/meta.yaml

```
@@ -0,0 +1,40 @@
+{% set name = "aioserial" %}
+{% set version = "1.1.3" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: "291357a062d537d2122cdc95fe2be2b74eaf310b385b067b70da3cf8b316ae7e"
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
+  noarch: python 
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
+  run:
+    - pyserial
+    - python >=3.6
+
+test:
+  imports:
+    - aioserial
+
+about:
+  home: http://github.com/changyuheng/aioserial
+  license: MPL-3.0
```

The license is ``MPL-2.0``
```suggestion
  license: MPL-2.0
```